### PR TITLE
Code format cleanup

### PR DESCRIPTION
**Pull request by rnystrom**
Cleaning up some inconsistencies that I found whilst digging.

### CODE REVIEW

File: AsyncDisplayKit/Details/ASFlowLayoutController.mm [hunk 1]
```diff
@@ -105,8 +105,8 @@ - (NSSet *)workingRangeIndexPathsForScrolling:(enum ASScrollDirection)scrollDire
   if (_layoutDirection == ASFlowLayoutDirectionHorizontal) {
     ASDisplayNodeAssert(scrollDirection == ASScrollDirectionNone || scrollDirection == ASScrollDirectionLeft || scrollDirection == ASScrollDirectionRight, @"Invalid scroll direction");
 
-    CGFloat backScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.leadingBufferScreenfuls : self.tuningParameters.trailingBufferScreenfuls;
-    CGFloat frontScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.trailingBufferScreenfuls : self.tuningParameters.leadingBufferScreenfuls;
+    int backScreens = scrollDirection == ASScrollDirectionLeft ? self.tuningParameters.leadingBufferScreenfuls : self.tuningParameters.trailingBufferScreenfuls;
```
Comment (secretiverhyme) [inline AsyncDisplayKit/Details/ASFlowLayoutController.mm]: this is a CGFloat as of 38f8e63b01e8a7565d854b49b54e47459c98d0b8, no change necessary =]

Comment (secretiverhyme) [inline AsyncDisplayKit/Details/ASFlowLayoutController.mm]: (also, for integral values, make sure to use NSInteger)

Comment (rnystrom) [inline AsyncDisplayKit/Details/ASFlowLayoutController.mm]: Ah ok! I'll change [these](https://github.com/facebook/AsyncDisplayKit/blob/master/AsyncDisplayKit/Details/ASFlowLayoutController.mm#L116-117) instead :smile:

**Comment by rnystrom** [comment]
Take 2 :grin:

**Comment by secretiverhyme** [comment]
Looks good.  Want to rebase against master?  GitHub reports merge conflicts.

**Comment by rnystrom** [comment]
There we go :)

**Pull request merged**